calico_node/startup/startup.go

```diff
@@ -420,13 +450,16 @@ func ipv6Supported() bool {
 	return supported
 }
 
-// createIPPool creates an IP pool using the specified CIDR string.  This
+// createIPPool creates an IP pool using the specified CIDR.  This
 // method is a no-op if the pool already exists.
-func createIPPool(client *client.Client, cs string) {
-	_, cidr, _ := net.ParseCIDR(cs)
+func createIPPool(client *client.Client, cidr *net.IPNet, ipip bool) {
 	version := cidr.Version()
 
-	log.Info("Ensure default IPv%d pool is created", version)
+	ipipEnabled := "without IPIP"
+	if ipip {
+		ipipEnabled = "with IPIP enabled"
+	}
+	log.Info("Ensure default IPv%d pool is created %s", version, ipipEnabled)
```

Comment:
I think you could just simplify this into the log line.  e.g

`"Ensure default IPv%d pool is created. IPIP enabled: %s", version, ipipEnabled`